### ISSUE TITLE
user-guide: add "folders" way of experimentation

### ISSUE THREAD

**Issue by efiop** [iterative/dvc.org#159]
Currently, we only mention branches as a way of experimentation (in Get Started), but it is not ideal and we've seen users use folders as an alternative. Need to mention it in our docs.

**Comment by shcheklein** [comment]
I think we should keep a single way in get started, mention other ways in user-guide and make a link to it. I'm switching get started to use tags now to avoid merging and other complicated ugly stuff.

**Comment by dashohoxha** [comment]
Maybe we can make a use-case for this.

Context:
- https://discordapp.com/channels/485586884165107732/485586884165107734/614145641831989248
- https://discordapp.com/channels/485586884165107732/485586884165107734/614506424420270080

This may also be related: https://github.com/iterative/dvc.org/issues/72

**Comment by dashohoxha** [comment]
Related: https://github.com/iterative/dvc/issues/2324

**Comment by dashohoxha** [comment]
The last step of this lesson mentions something about using different folders for different pipelines:
https://katacoda.com/dvc/courses/basics/pipelines
But I am not sure if this is enough to close this issue.

**Comment by efiop** [comment]
@dashohoxha Feels like it would be suitable to describe that more formally, hence why I was mentioning "user guide" and not "get-started" or "tutorial".

**Comment by dashohoxha** [comment]
@efiop Right. But some things are better explained with a working example. It may also be easier for the users to understand it. And an example can also be complemented by formal and generalized descriptions.

**Comment by jorgeorpinel** [comment]
> general **Managing Experiments** use case or user guide section. It can include different approaches - folders, tags, branches, commits, etc.

I reviewed all docs and he term "experiment" is not mentioned in the User Guide at all, and only once in Use Cases (in Shared Development Server). Otherwise we mainly talk about experiments in command references and tutorials. So it seems this new **Managing Experiments** is still a valid enhancement. Please note we should use the term "directory" instead of "folder" though.

**Comment by dashohoxha** [comment]
> So it seems this new **Managing Experiments** is still a valid enhancement.

@jorgeorpinel are you working on it?
I am asking so that we don't make a conflict with #816.

**Comment by jorgeorpinel** [comment]
I'm not. I was just reviewing the validity of some older issues.
But if #816 addresses this, maybe one of the 2 should be closed?

**Comment by dashohoxha** [comment]
> But if #816 addresses this, maybe one of the 2 should be closed?

I think that they overlap but are not quite the same. We can close them both with this PR: https://github.com/iterative/dvc.org/pull/828

**Comment by jorgeorpinel** [comment]
Closing in favor of #816
